app/Libraries/Search/ScoreSearchParams.php

```diff
@@ -24,6 +24,7 @@ class ScoreSearchParams extends SearchParams
     public bool $excludeConverts = false;
     public ?array $excludeMods = null;
     public ?bool $isLegacy = null;
+    public bool $excludeScoresWithoutPP = false;
```

Comment:
I would think the scores part would be redundant in this context...

Reply:
I think so but this line should also be renamed 🤔

And I think osu-web uses `Pp` everywhere... (when not lowercased)